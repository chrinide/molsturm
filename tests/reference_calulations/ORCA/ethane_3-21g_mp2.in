! bohrs mp2 EXTREMESCF nofrozencore 3-21G nopop noprintmos

* xyz 0 1
C    0.11773122601810967  2.706579152570035e-07  -0.36282169701109324
C    0.8516848710413312  -3.5297816510288056e-07  2.420736070352015
H    2.9018095114074667  -2.3439803917193614e-06  2.656181084227907
H    0.11360588178037678  -1.6647921043201028  3.3913802657097327
H    0.1136091723264787  1.6647931911034923  3.39137973979706
H    -1.9323875392990797  3.854368570695653e-06  -0.5983078420216195
H    0.8558352286352563  1.66479071436795  -1.3334507521041237
H    0.8558296748964446  -1.664793229221158  -1.3334498295870683
*
