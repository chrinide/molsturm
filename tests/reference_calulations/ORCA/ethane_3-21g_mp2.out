
                                 *****************
                                 * O   R   C   A *
                                 *****************

           --- An Ab Initio, DFT and Semiempirical electronic structure package ---

                  #######################################################
                  #                        -***-                        #
                  #  Department of molecular theory and spectroscopy    #
                  #              Directorship: Frank Neese              #
                  # Max Planck Institute for Chemical Energy Conversion #
                  #                  D-45470 Muelheim/Ruhr              #
                  #                       Germany                       #
                  #                                                     #
                  #                  All rights reserved                #
                  #                        -***-                        #
                  #######################################################


                         Program Version 4.0.0 -  RELEASE  -


 With contributions from (in alphabetic order):
   Daniel Aravena         : Magnetic Properties
   Michael Atanasov       : Ab Initio Ligand Field Theory
   Ute Becker             : Parallelization
   Martin Brehm           : Molecular dynamics
   Dmytro Bykov           : SCF Hessian
   Vijay G. Chilkuri      : MRCI spin determinant printing
   Dipayan Datta          : RHF DLPNO-CCSD density
   Achintya Kumar Dutta   : EOM-CC, STEOM-CC
   Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI
   Yang Guo               : DLPNO-NEVPT2, CIM, IAO-localization
   Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods
   Lee Huntington         : MR-EOM, pCC
   Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM
   Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density
   Simone Kossmann        : Meta GGA functionals, TD-DFT gradient, OOMP2, MP2 Hessian
   Martin Krupicka        : AUTO-CI
   Dagmar Lenk            : GEPOL surface
   Dimitrios Liakos       : Extrapolation schemes; parallel MDCI
   Dimitrios Manganas     : ROCIS; embedding schemes
   Dimitrios Pantazis     : SARC Basis sets
   Taras Petrenko         : DFT Hessian,TD-DFT gradient, ASA, ECA, R-Raman, ABS, FL, XAS/XES, NRVS
   Peter Pinski           : DLPNO-MP2
   Christoph Reimann      : Effective Core Potentials
   Marius Retegan         : Local ZFS, SOC
   Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples
   Tobias Risthaus        : Range-separated hybrids, TD-DFT gradient, RPA, STAB
   Michael Roemelt        : Restricted open shell CIS
   Masaaki Saitow         : Open-shell DLPNO
   Barbara Sandhoefer     : DKH picture change effects
   Kantharuban Sivalingam : CASSCF convergence, NEVPT2, FIC-MRCI
   Georgi Stoychev        : AutoAux
   Boris Wezisla          : Elementary symmetry handling
   Frank Wennmohs         : Technical directorship


 We gratefully acknowledge several colleagues who have allowed us to
 interface, adapt or use parts of their codes:
   Stefan Grimme, W. Hujo, H. Kruse,             : VdW corrections, initial TS optimization,
                  C. Bannwarth                     DFT functionals, gCP, sTDA/sTD-DF
   Ed Valeev                                     : LibInt (2-el integral package), F12 methods
   Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG
   Ulf Ekstrom                                   : XCFun DFT Library
   Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
   Jiri Pittner, Ondrej Demel                    : Mk-CCSD
   Frank Weinhold                                : gennbo (NPA and NBO analysis)
   Christopher J. Cramer and Donald G. Truhlar   : smd solvation model


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines


Your calculation utilizes the basis: 3-21G
   H-Ne       : J. S. Binkley, J. A. Pople and W. J. Hehre, J. Am. Chem. Soc. 102 939 (1980).
   Na-Ar      : M. S. Gordon, J. S. Binkley, J. A. Pople, W. J. Pietro and W.J. Hehre, J. Am. Chem. Soc. 104, 2797 (1983).
   K-Ca, Ga-Kr: K. D. Dobbs and W. J. Hehre, J. Comput. Chem. 7, 359 (1986).
   Sc-Zn      : K. D. Dobbs and W. J. Hehre, J. Comput. Chem. 8, 861 (1987).
   Y-Cd       : K. D. Dobbs and W. J. Hehre, J. Comput. Chem. 8, 880 (1987).

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================

Warning: TCutStore was < 0. Adjusted to Thresh (uncritical)

WARNING: TCut can not be >=Thresh
  ===> : TCut is set to 0.01*Thresh

WARNING: Post HF methods need fully converged wavefunctions
  ===> : Setting SCFConvForced true
         You can overwrite this default with %scf ConvForced false 


INFO   : the flag for use of LIBINT has been found!

================================================================================
                                       INPUT FILE
================================================================================
NAME = ethane_3-21g_mp2.in
|  1> ! bohrs mp2 EXTREMESCF nofrozencore 3-21G nopop noprintmos
|  2> 
|  3> * xyz 0 1
|  4> C    0.11773122601810967  2.706579152570035e-07  -0.36282169701109324
|  5> C    0.8516848710413312  -3.5297816510288056e-07  2.420736070352015
|  6> H    2.9018095114074667  -2.3439803917193614e-06  2.656181084227907
|  7> H    0.11360588178037678  -1.6647921043201028  3.3913802657097327
|  8> H    0.1136091723264787  1.6647931911034923  3.39137973979706
|  9> H    -1.9323875392990797  3.854368570695653e-06  -0.5983078420216195
| 10> H    0.8558352286352563  1.66479071436795  -1.3334507521041237
| 11> H    0.8558296748964446  -1.664793229221158  -1.3334498295870683
| 12> *
| 13> 
| 14>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  C      0.062301    0.000000   -0.191997
  C      0.450692   -0.000000    1.280998
  H      1.535571   -0.000001    1.405590
  H      0.060118   -0.880970    1.794641
  H      0.060119    0.880971    1.794641
  H     -1.022575    0.000002   -0.316611
  H      0.452888    0.880969   -0.705632
  H      0.452886   -0.880971   -0.705631

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 C     6.0000    0    12.011    0.117731    0.000000   -0.362822
   1 C     6.0000    0    12.011    0.851685   -0.000000    2.420736
   2 H     1.0000    0     1.008    2.901810   -0.000002    2.656181
   3 H     1.0000    0     1.008    0.113606   -1.664792    3.391380
   4 H     1.0000    0     1.008    0.113609    1.664793    3.391380
   5 H     1.0000    0     1.008   -1.932388    0.000004   -0.598308
   6 H     1.0000    0     1.008    0.855835    1.664791   -1.333451
   7 H     1.0000    0     1.008    0.855830   -1.664793   -1.333450

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 C      0   0   0     0.000000000000     0.00000000     0.00000000
 C      1   0   0     1.523339498191     0.00000000     0.00000000
 H      2   1   0     1.092010142899   111.32263863     0.00000000
 H      2   1   3     1.092009749030   111.32324942   240.00019151
 H      2   1   3     1.092009760669   111.32323753   119.99980034
 H      1   2   3     1.092009537819   111.32379179   180.00004746
 H      1   2   3     1.092010156636   111.32261465   300.00035265
 H      1   2   3     1.092010180547   111.32260828    59.99973343

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 C      0   0   0     0.000000000000     0.00000000     0.00000000
 C      1   0   0     2.878694460566     0.00000000     0.00000000
 H      2   1   0     2.063600105544   111.32263863     0.00000000
 H      2   1   3     2.063599361239   111.32324942   240.00019151
 H      2   1   3     2.063599383234   111.32323753   119.99980034
 H      1   2   3     2.063598962107   111.32379179   180.00004746
 H      1   2   3     2.063600131503   111.32261465   300.00035265
 H      1   2   3     2.063600176687   111.32260828    59.99973343

---------------------
BASIS SET INFORMATION
---------------------
There are 2 groups of distinct atoms

 Group   1 Type C   : 6s3p contracted to 3s2p pattern {321/21}
 Group   2 Type H   : 3s contracted to 2s pattern {21}

Atom   0C    basis set group =>   1
Atom   1C    basis set group =>   1
Atom   2H    basis set group =>   2
Atom   3H    basis set group =>   2
Atom   4H    basis set group =>   2
Atom   5H    basis set group =>   2
Atom   6H    basis set group =>   2
Atom   7H    basis set group =>   2
------------------------------------------------------------------------------
                           ORCA GTO INTEGRAL CALCULATION
------------------------------------------------------------------------------

                         BASIS SET STATISTICS AND STARTUP INFO

 # of primitive gaussian shells          ...   36
 # of primitive gaussian functions       ...   48
 # of contracted shells                  ...   22
 # of contracted basis functions         ...   30
 Highest angular momentum                ...    1
 Maximum contraction depth               ...    3
 Integral package used                   ... LIBINT
 Integral threshhold            Thresh   ...  3.000e-16
 Primitive cut-off              TCut     ...  3.000e-18


------------------------------ INTEGRAL EVALUATION ----------------------------


 * One electron integrals 
 Pre-screening matrix                    ... done
 Shell pair data                         ... done (   0.000 sec)

-------------------------------------------------------------------------------
                                 ORCA SCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 Ab initio Hamiltonian  Method          .... Hartree-Fock(GTOs)


General Settings:
 Integral files         IntName         .... ethane_3-21g_mp2
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....   18
 Basis Dimension        Dim             ....   30
 Nuclear Repulsion      ENuc            ....     42.2882494112 Eh

Convergence Acceleration:
 DIIS                   CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 Newton-Raphson         CNVNR           .... off
 SOSCF                  CNVSOSCF        .... on
   Start iteration      SOSCFMaxIt      ....   150
   Startup grad/error   SOSCFStart      ....  0.003300
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000
 Fernandez-Rico         CNVRico         .... off

SCF Procedure:
 Maximum # iterations   MaxIter         ....   125
 SCF integral mode      SCFMode         .... Direct
   Integral package                     .... LIBINT
 Reset frequeny         DirectResetFreq ....    20
 Integral Threshold     Thresh          ....  3.000e-16 Eh
 Primitive CutOff       TCut            ....  3.000e-18 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... All-Criteria
 Convergence forced     ConvForced      .... 1
 Energy Change          TolE            ....  1.000e-14 Eh
 Max Density Change     TolMaxP         ....  1.000e-14
 RMS Density Change     TolRMSP         ....  1.000e-14
 Orbital Gradient       TolG            ....  1.000e-09
 Orbital Rotation angle TolX            ....  1.000e-09
 DIIS Error             TolErr          ....  1.000e-14


Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 1.866e-02
Time for diagonalization                   ...    0.000 sec
Threshold for overlap eigenvalues          ... 1.000e-08
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.000 sec
Total time needed                          ...    0.000 sec

-------------------
DFT GRID GENERATION
-------------------

General Integration Accuracy     IntAcc      ...  4.340
Radial Grid Type                 RadialGrid  ... Gauss-Chebyshev
Angular Grid (max. acc.)         AngularGrid ... Lebedev-110
Angular grid pruning method      GridPruning ... 3 (G Style)
Weight generation scheme         WeightScheme... Becke
Basis function cutoff            BFCut       ...    3.0000e-16
Integration weight cutoff        WCut        ...    1.0000e-14
Grids for H and He will be reduced by one unit

# of grid points (after initial pruning)     ...   8076 (   0.0 sec)
# of grid points (after weights+screening)   ...   7742 (   0.0 sec)
nearest neighbour list constructed           ...    0.0 sec
Grid point re-assignment to atoms done       ...    0.0 sec
Grid point division into batches done        ...    0.0 sec
Reduced shell lists constructed in    0.0 sec

Total number of grid points                  ...     7742
Total number of batches                      ...      125
Average number of points per batch           ...       61
Average number of grid points per atom       ...      968
Average number of shells per batch           ...    19.98 (90.84%)
Average number of basis functions per batch  ...    27.63 (92.12%)
Average number of large shells per batch     ...    18.46 (92.37%)
Average number of large basis fcns per batch ...    25.92 (93.80%)
Maximum spatial batch extension              ...  22.36, 25.60, 20.71 au
Average spatial batch extension              ...   4.73,  4.56,  4.45 au

Time for grid setup =    0.050 sec

------------------------------
INITIAL GUESS: MODEL POTENTIAL
------------------------------
Loading Hartree-Fock densities                     ... done
Calculating cut-offs                               ... done
Setting up the integral package                    ... done
Initializing the effective Hamiltonian             ... done
Starting the Coulomb interaction                   ... done (   0.0 sec)
Reading the grid                                   ... done
Mapping shells                                     ... done
Starting the XC term evaluation                    ... done (   0.1 sec)
Transforming the Hamiltonian                       ... done (   0.0 sec)
Diagonalizing the Hamiltonian                      ... done (   0.0 sec)
Back transforming the eigenvectors                 ... done (   0.0 sec)
Now organizing SCF variables                       ... done
                      ------------------
                      INITIAL GUESS DONE (   0.2 sec)
                      ------------------
--------------
SCF ITERATIONS
--------------
ITER       Energy         Delta-E        Max-DP      RMS-DP      [F,P]     Damp
               ***  Starting incremental Fock matrix formation  ***
  0    -78.6683950108   0.000000000000 0.03978023  0.00517440  0.2069207 0.7000
  1    -78.7140047019  -0.045609691118 0.03525744  0.00430500  0.1341688 0.7000
                               ***Turning on DIIS***
  2    -78.7411205711  -0.027115869175 0.07945807  0.00946833  0.0796781 0.0000
  3    -78.5873259366   0.153794634519 0.01445894  0.00209027  0.0307674 0.0000
  4    -78.7724486354  -0.185122698839 0.00475914  0.00044530  0.0038183 0.0000
                      *** Initiating the SOSCF procedure ***
                           *** Shutting down DIIS ***
                      *** Re-Reading the Fockian *** 
                      *** Removing any level shift *** 
ITER      Energy       Delta-E        Grad      Rot      Max-DP    RMS-DP
  5    -78.79086214  -0.0184135053  0.000921  0.000921  0.002264  0.000183
               *** Restarting incremental Fock matrix formation ***
  6    -78.79353200  -0.0026698593  0.000274  0.000348  0.000823  0.000069
  7    -78.79353246  -0.0000004618  0.000095  0.000183  0.000271  0.000029
  8    -78.79353251  -0.0000000511  0.000013  0.000009  0.000019  0.000002
  9    -78.79353251  -0.0000000013  0.000004  0.000002  0.000007  0.000001
 10    -78.79353251  -0.0000000001  0.000000  0.000000  0.000001  0.000000
 11    -78.79353251  -0.0000000000  0.000000  0.000000  0.000000  0.000000
                          ***Energy convergence achieved***
 12    -78.79353251   0.0000000000  0.000000  0.000000  0.000000  0.000000
 13    -78.79353251  -0.0000000000  0.000000  0.000000  0.000000  0.000000
                           ***RMSP convergence achieved***
                           ***MAXP convergence achieved***
                       ***Gradient convergence achieved***
                    ***Orbital Rotation convergence achieved***
 14    -78.79353251  -0.0000000000  0.000000  0.000000  0.000000  0.000000
                          ***Energy convergence achieved***
              ***Rediagonalizing the Fockian in SOSCF/NRSCF***

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER  15 CYCLES          *
               *****************************************************


----------------
TOTAL SCF ENERGY
----------------

Total Energy       :          -78.79353251 Eh           -2144.08102 eV

Components:
Nuclear Repulsion  :           42.28824941 Eh            1150.72177 eV
Electronic Energy  :         -121.08178193 Eh           -3294.80279 eV
One Electron Energy:         -188.47879917 Eh           -5128.76887 eV
Two Electron Energy:           67.39701724 Eh            1833.96608 eV

Virial components:
Potential Energy   :         -157.40404948 Eh           -4283.18194 eV
Kinetic Energy     :           78.61051697 Eh            2139.10092 eV
Virial Ratio       :            2.00232813


---------------
SCF CONVERGENCE
---------------

  Last Energy change         ...    0.0000e+00  Tolerance :   1.0000e-14
  Last MAX-Density change    ...    0.0000e+00  Tolerance :   1.0000e-14
  Last RMS-Density change    ...    0.0000e+00  Tolerance :   1.0000e-14
  Last Orbital Gradient      ...    9.9135e-10  Tolerance :   1.0000e-09
  Last Orbital Rotation      ...    0.0000e+00  Tolerance :   1.0000e-09

             **** THE GBW FILE WAS UPDATED (ethane_3-21g_mp2.gbw) ****
             **** DENSITY FILE WAS UPDATED (ethane_3-21g_mp2.scfp.tmp) ****
             **** ENERGY FILE WAS UPDATED (ethane_3-21g_mp2.en.tmp) ****
----------------
ORBITAL ENERGIES
----------------

  NO   OCC          E(Eh)            E(eV) 
   0   2.0000     -11.153926      -303.5138 
   1   2.0000     -11.153668      -303.5067 
   2   2.0000      -1.021042       -27.7840 
   3   2.0000      -0.842799       -22.9337 
   4   2.0000      -0.598213       -16.2782 
   5   2.0000      -0.598213       -16.2782 
   6   2.0000      -0.510102       -13.8806 
   7   2.0000      -0.482209       -13.1216 
   8   2.0000      -0.482209       -13.1216 
   9   0.0000       0.278467         7.5775 
  10   0.0000       0.335282         9.1235 
  11   0.0000       0.335282         9.1235 
  12   0.0000       0.340851         9.2750 
  13   0.0000       0.370603        10.0846 
  14   0.0000       0.370604        10.0846 
  15   0.0000       0.432701        11.7744 
  16   0.0000       0.909415        24.7464 
  17   0.0000       0.909416        24.7465 
  18   0.0000       0.966257        26.2932 
  19   0.0000       1.084323        29.5059 
  20   0.0000       1.092219        29.7208 
  21   0.0000       1.092221        29.7208 
  22   0.0000       1.287797        35.0428 
  23   0.0000       1.360790        37.0290 
  24   0.0000       1.360791        37.0290 
  25   0.0000       1.363882        37.1131 
  26   0.0000       1.363882        37.1131 
  27   0.0000       1.443343        39.2754 
  28   0.0000       1.935394        52.6647 
  29   0.0000       2.297531        62.5190 
-------
TIMINGS
-------

Total SCF time: 0 days 0 hours 0 min 1 sec 

Total time                  ....       1.863 sec
Sum of individual times     ....       1.859 sec  ( 99.8%)

Fock matrix formation       ....       1.654 sec  ( 88.8%)
Diagonalization             ....       0.002 sec  (  0.1%)
Density matrix formation    ....       0.000 sec  (  0.0%)
Population analysis         ....       0.000 sec  (  0.0%)
Initial guess               ....       0.143 sec  (  7.7%)
Orbital Transformation      ....       0.000 sec  (  0.0%)
Orbital Orthonormalization  ....       0.000 sec  (  0.0%)
DIIS solution               ....       0.001 sec  (  0.0%)
SOSCF solution              ....       0.010 sec  (  0.5%)

------------------------------------------------------------------------------
                                ORCA  MP2 
------------------------------------------------------------------------------


----------
MP2 ENERGY (disk based algorithm)
----------

Dimension of the basis                    ...   30
Memory devoted to MP2                     ...  256 MB   
Data format for buffers                   ... DOUBLE
Compression type for matrix containers    ... UNCOMPRESSED

-------------------------------
PARTIAL EXCHANGE TRANSFORMATION
-------------------------------

Transformation type                        ... one-operator
Generation of integrals (i,mue|j,nue)      ... ON
Generation of integrals (mue,kappa|nue,tau)... OFF
Generation of integrals (i,mue|a,nue)      ... OFF
Dimension of the basis                     ...   30
Number of internal MOs                     ...    9 (   0-   8)
Pair cutoff                                ... 3.000e-18 Eh
Number of AO pairs in the trafo            ...  465
Total Number of distinct AO pairs          ...  465
Memory devoted for trafo                   ...  256.0 MB 
Max. Number of MO pairs treated together   ... 37282      
Number Format for Storage                  ... Double (8 Byte)
Integral package used                      ... LIBINT

Starting integral evaluation:
    ... done with AO integral generation
Closing buffer AOK     ( 0.00 GB; CompressionRatio= 0.98)
Collecting buffer AOK 
Number of MO pairs included in the trafo   ...   45
------------------------
SORTING OF (i,mue|j,nue)
------------------------

IBATCH = 1 of  1
Closing buffer KAO     ( 0.00 GB; CompressionRatio= 1.00)
Collecting buffer KAO 
N(AO-Batches) Done                         ...    122449 
N(AO-Batches) Skipped                      ...         3 
N(IJ)-pairs Skipped                        ...         0 
TOTAL TIME for half transformation         ...     0.055 sec
AO-integral generation                     ...     0.050 sec
Half transformation                        ...     0.003 sec
K-integral sorting                         ...     0.001 sec

Finished integral transformation - now doing MP2 part

OPERATOR COMBINATION   0   0: ij=(  0..  8,  0..  8)
  Internal MO   2
  Internal MO   5
  Internal MO   8
-----------------------------------------------
 MP2 CORRELATION ENERGY   :     -0.191344399 Eh
-----------------------------------------------


-------
TIMINGS
-------
Total time                :    0.122 sec
Integral trafo            :    0.055 sec ( 44.9%)
K(i,j)                    :    0.001 sec (  1.2%)
T(i,j)                    :    0.000 sec (  0.0%)

---------------------------------------
MP2 TOTAL ENERGY:      -78.984876913 Eh
---------------------------------------

-------------------------   --------------------
FINAL SINGLE POINT ENERGY       -78.984876912917
-------------------------   --------------------


                            ***************************************
                            *     ORCA property calculations      *
                            ***************************************


  WARNING: The energy has been calculated at the MP2 level but the densities
           used in the property calculations are still SCF densities
           MP2 response densities have not been calculated 
           use %mp2 Density relaxed end
           or  %mp2 Density unrelaxed end

                                    ---------------------
                                    Active property flags
                                    ---------------------
   (+) Dipole Moment


------------------------------------------------------------------------------
                       ORCA ELECTRIC PROPERTIES CALCULATION
------------------------------------------------------------------------------

Dipole Moment Calculation                       ... on
Quadrupole Moment Calculation                   ... off
Polarizability Calculation                      ... off
GBWName                                         ... ethane_3-21g_mp2.gbw
Electron density file                           ... ethane_3-21g_mp2.scfp.tmp

-------------
DIPOLE MOMENT
-------------
                                X             Y             Z
Electronic contribution:     -0.00002      -0.00000       0.00000
Nuclear contribution   :      0.00002       0.00000      -0.00000
                        -----------------------------------------
Total Dipole Moment    :      0.00000       0.00000      -0.00000
                        -----------------------------------------
Magnitude (a.u.)       :      0.00000
Magnitude (Debye)      :      0.00001


Timings for individual modules:

Sum of individual times         ...        2.175 sec (=   0.036 min)
GTO integral calculation        ...        0.167 sec (=   0.003 min)   7.7 %
SCF iterations                  ...        1.871 sec (=   0.031 min)  86.0 %
MP2 module                      ...        0.137 sec (=   0.002 min)   6.3 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 2 seconds 265 msec
